skycoin/skywire
# Additional features for configuring skysocks-client in the manager

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Maybe Fixes #314

 Changes:	
- Now the configuration modal window for the skysocks-client  app shows the list of nodes obtained from the proxy discovery service. Options for filtering the list were added.
- The history has been improved.
- Various other small improvements.

Notes:
- This PR includes the changes made in #315
- The PR still lack pagination options.
- The PR is currently connecting to a local instance of the proxy discovery service.
- The list obtained from the proxy discovery service does not include information about the auth state, so it was not possible to include that information or to use it for filtering.

How to test this PR:
Use the `make integration-run-proxy` command of the `skywire-services` repo and check the configuration options of the skysocks-client app.
